### PR TITLE
Fixed #4167 Integrate lookbook for view components

### LINKED ISSUE #4167: Integrate lookbook for view components
Refer: https://github.com/ViewComponent/lookbook

### PR DESCRIPTION
**Pull request by shakthi2003-ice**
Fixed #4167

Fixes #

#### Describe the changes you have made in this PR -
I have integrated LookBook for Viewcomponent

### Screenshots of the changes (If any) -

Screenshot 2023-10-15 at 5 12 19 PM

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by tanmoysrt** [comment]
@shakthi2003-ice Please provide a valid PR title and description.
And you must be assigned to the issue before raising any PR.

Please follow that next time, also get assigned yourself to the issue first before reviewing.

**Comment by shakthi2003-ice** [comment]
]

**Comment by shakthi2003-ice** [comment]
@tanmoysrt I already requested to assign me this issue as it has not been assigned to anyone. But I didn't get any response so i started working on the issue. Extremely sorry. Will make sure that it won't happen again

**Comment by tachyons** [comment]
Could you add at least one component on this page? https://github.com/CircuitVerse/CircuitVerse/tree/master/app/component

**Comment by shakthi2003-ice** [comment]
> Could you add at least one component on this page? https://github.com/CircuitVerse/CircuitVerse/tree/master/app/component

@tachyons sure

File: Gemfile [hunk 1]
```diff
@@ -213,3 +213,7 @@ gem "opentelemetry-instrumentation-action_pack"
 gem "opentelemetry-instrumentation-action_view"
 
 gem "maintenance_tasks", "~> 2.3"
+
+group :development do
```
Comment (niladrix719) [inline Gemfile:217]: there is already a `group :development do` here: https://github.com/CircuitVerse/CircuitVerse/blob/0762999968fc96c632edd6a3fe5beca776b8c416/Gemfile#L154
you could have added the gem there

File: config/routes.rb [hunk 2]
```diff
@@ -237,3 +237,8 @@
   end
 end
 # rubocop:enable Metrics/BlockLength
+Rails.application.routes.draw do
```
Comment (niladrix719) [inline config/routes.rb:240]: there is already a `Rails.application.routes.draw do` here: https://github.com/CircuitVerse/CircuitVerse/blob/0762999968fc96c632edd6a3fe5beca776b8c416/config/routes.rb#L4, you can add it here

**Review by niladrix719 — CHANGES_REQUESTED** [review]

**Comment by niladrix719** [comment]
@shakthi2003-ice i think the gem "actioncable" is also missing, can you make it so that you can Enable live UI updates on the components - https://github.com/CircuitVerse/CircuitVerse/tree/master/app/component

**Pull request closed without merging**